app/components/account/AccountContainer.js

```diff
@@ -12,96 +12,65 @@ import moment from 'moment';
 
 // -=-=-=-=-=-= COMPONENT =-=-=-=-=-=-
 
-export class AccountInfo extends Component {
-
-  constructor(props) {
-    super(props)
-    this.state = {}
-  }
-
-  render() {
-
-    const {user} = this.props;
-
-    return (
-
-      <div id="account-info" className="col-xs-12" > 
-        <Helmet title="Account Info" />
-          <h1 className="page-header">Account Info</h1>
-          <Paper style={{ backgroundColor: background, disabledTextColor: primary, padding:'15px' }} zDepth={2} rounded={false} >
-            <div className="row">
-              <div className="col-xs-12 col-sm-6 col-lg-offset-2 col-lg-8">
-                <StepsTextField
-                  floatingLabelText="First Name"
-                  value={user.first_name}
-                  fullWidth={ true }
-                  />
-              </div>
-              <div className="col-xs-12 col-sm-6 col-lg-offset-2 col-lg-8">
-                <StepsTextField
-                  floatingLabelText="Last Name"
-                  value={user.last_name}
-                  fullWidth={ true }
-                   />
-              </div>
+const AccountInfo = ({ user }) => (
+  <div id="account-info" className="col-xs-12" > 
```

Comment:
You made me conform to flex-box with our component overlap! Was a good learning experience. Code looks good to me. Moving on to click testing.